stdlib/multiprocessing/context.pyi

```diff
@@ -79,14 +80,22 @@ class BaseContext:
     @overload
     def RawArray(self, typecode_or_type: str, size_or_initializer: int | Sequence[Any]) -> Any: ...
     @overload
-    def Value(self, typecode_or_type: type[_CT], *args: Any, lock: Literal[False]) -> Synchronized[_CT]: ...
+    def Value(
+        self, typecode_or_type: _SimpleCData[_T], *args: Any, lock: Literal[True] | _LockLike = True
+    ) -> Synchronized[_T]: ...
     @overload
-    def Value(self, typecode_or_type: type[_CT], *args: Any, lock: Literal[True] | _LockLike = True) -> Synchronized[_CT]: ...
+    def Value(self, typecode_or_type: _CT, *args: Any, lock: Literal[False]) -> Synchronized[_CT]: ...
+    @overload
+    def Value(self, typecode_or_type: _CT, *args: Any, lock: Literal[True] | _LockLike = True) -> Synchronized[_CT]: ...
```

Comment:
You removed the `type` "wrapper" here, while `Array` still has it, was this intentional?

Also, could you add some test cases to `test_cases/stdlib`? I think the multiprocessing stubs are sufficiently complex to warrant some.

Reply:
this was intentional and a misstake, which is why i re-added them now. I agree with test-cases, let's see what I can come up with.